### ISSUE TITLE
BEAT Training Data

### ISSUE THREAD

**Issue by Symbolzzz** [YoungSeng/DiffuseStyleGesture#42]
How much data is used to train the pre-trained BEAT model provided in Google Drive? Is it trained using all the data from speakers 2 and 10 in the original BEAT dataset? Or is it trained using only one file in `./BEAT/train?`

**Comment by YoungSeng** [comment]
Yes, I remember I just happened to use data from two speakers for training the BEAT model, but I didn’t use BEAT in the paper. For DiffuseStyleGesture, I used `ZEGGS`, and for DiffuseStyleGesture+, I used `Trinity`.
